components/feature/app-links/src/main/java/mozilla/components/feature/app/links/RedirectDialogFragment.kt

```diff
@@ -22,6 +22,13 @@ abstract class RedirectDialogFragment : DialogFragment() {
      */
     var onConfirmRedirect: () -> Unit = {}
 
+    /**
+     * A callback to trigger a URL load, call it when you are ready load the URL in the current session.
```

Comment:
nit: I would rephrase this. The callback isn't specifically to trigger a load, it's invoked when the user cancels / dismisses the dialog. re: "to trigger a URL load when you are...".

Reply:
Sure, I'll rename.